### DISCUSSION TITLE
Feature in Adobe Sumatra doesnt have yet

### DISCUSSION THREAD

**Discussion by Demlar** [sumatrapdfreader/sumatrapdf#4383] [Ideas]
This primarily happens  when using altium pdf export. I honestly don't know what this feature is even called.
if I left click I see this on adobe.

currently sumatra doesnt support the same thing, if I left click I get nothing. Right click looks like this

What I am doing here is left clicking the component, I have circled it below.

**Comment by kjk** [comment]
Please open an issue (https://github.com/sumatrapdfreader/sumatrapdf/issues) and provide a test document.

**Comment by Demlar** [comment]
Will do I'll try and find something generic.

**Reply by GitHubRulesOK** [reply]
This has already been asked and is not generically possible in MuPDF engine it is a sophisticated alternative use of an Adobe Ecma scripted Pop-up Menu system. It might look easy to emulate but when trying to play with it in acrobat its a challenge due to different user application methodologies.

It is fairly niche to the electronics market as it allows for floating Integrated Circuitry. Let me see which open issue covers these modals

**Reply by GitHubRulesOK** [reply]
Ahh here it is https://github.com/sumatrapdfreader/sumatrapdf/issues/1198
